src/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -4,10 +4,21 @@
 
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
 using Internal.Runtime.CompilerServices;
 
+#if BIT64
```

Comment:
`BIT64` define works in CoreLib only today. It does not work in CoreFX- CoreFX is not built as bit-specific.

Reply:
Ah, god to know. Thanks. I'll revert to `(IntPtr)` then?